### DISCUSSION TITLE
Hyperlink that opens associated editor at given line and column number [Windows only]

### DISCUSSION THREAD

**Discussion by netcorefan1** [microsoft/terminal#15318] [Q&A] [answered]
Hello, 
the below code is supposed to open a file in VSCode at line 5 column 10: 
```Write-Output "`e]8;;vscode://file/C:/MyFile.py:5:10`e\This link will open 'MyFile.py' to line 5, col 10`e]8;;`e\"```

When I click the link I get:
```
This link type is currently not supported:
vscode://file/C:/MyFile.py:5:10
```
I saw several works in progress on the matter and I'm wondering if I'm doing something wrong or I should just wait for a new version in the winget/store repo (I suppose this is not yet supported even in 1.17.10234.0 preview).
Thanks

**Comment by j4james** [comment, marked as answer]
Support for additional URL types was added in PR #14993, which was merged in March, but that functionality hasn't been released yet. It should be included in the 1.18 preview release, and I think it might also be included in the 1.17 stable release, since the PR has been assigned to the 1.17 servicing pipeline.

**Reply by netcorefan1** [reply]
Many thanks @j4james ! That was all I needed to know!
